Detectors/EMCAL/workflow/src/digit-reader-workflow.cxx

```diff
@@ -0,0 +1,56 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include <string>
+#include <vector>
+#include "Framework/Variant.h"
+#include "Framework/ConfigParamSpec.h"
+#include "DataFormatsEMCAL/Digit.h"
+#include "DetectorsRaw/HBFUtilsInitializer.h"
+#include "EMCALWorkflow/PublisherSpec.h"
+#include "CommonUtils/ConfigurableParam.h"
+
+using namespace o2::framework;
+using namespace o2::emcal;
+
+// we need to add workflow options before including Framework/runDataProcessing
+void customize(std::vector<ConfigParamSpec>& workflowOptions)
+{
+  std::vector<ConfigParamSpec> options{{"disable-mc", VariantType::Bool, false, {"Do not propagate MC labels"}},
+                                       {"configKeyValues", VariantType::String, "", {"Semicolon separated key=value strings"}}};
```

Comment:
You missed here `o2::raw::HBFUtilsInitializer::addConfigOption(options);`

Reply:
Done, thanks for spotting!